Makefile

```diff
@@ -34,6 +35,11 @@ else
 GOBIN=$(shell go env GOBIN)
 endif
 
+# Image URL to use all building/pushing image targets
+VELA_IMAGE           ?= vela:latest
```

Comment:
where is it used?

Reply:
Considering #1392, I added it in advance. `VELA_IMAGE` can also be removed first

Reply:
Let's remove it cause it's confusion , we can add it at that time.

Reply:
Done